### PR TITLE
Bugfix rename function

### PR DESCRIPTION
**Pull request by vcward**

### CODE REVIEW

File: lib/series_data_adjustment.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ def last_value_date
   end
        
   def trim(start_date = get_last_incomplete_january, end_date = Time.now.to_date)
-    new_series_data = get_values_after_including(start_date, end_date)
+    new_series_data = get_values_after_including((Date.parse start_date.to_s), (Date.parse end_date.to_s))
```
Comment (jonpage) [inline lib/series_data_adjustment.rb]: I thought this change was made in #82

File: lib/series_seasonal_adjustment.rb [hunk 2]
```diff
@@ -5,10 +5,10 @@ def apply_seasonal_adjustment(factor_application)
     ns_series = Series.get ns_series_name
     raise SeasonalAdjustmentException.new if ns_series.nil?
     set_factors factor_application 
-    new_ns_values = ns_series.get_values_after last_demetra_datestring
+    new_ns_values = ns_series.get_values_after (Date.parse last_demetra_datestring)
     adjusted_data = {}
     new_ns_values.each do |datestring, value|
```
Comment (jonpage) [inline lib/series_seasonal_adjustment.rb]: We could rename 'datestring' to 'date' to reflect it is not supposed to be a String.

File: lib/series_seasonal_adjustment.rb [hunk 3]
```diff
@@ -32,12 +32,12 @@ def set_factors(factor_application)
     self.last_demetra_datestring = (self.frequency == "quarter" or self.frequency == "Q") ? self.get_last_complete_4th_quarter : self.get_last_complete_december
```
Comment (jonpage) [inline lib/series_seasonal_adjustment.rb]: `last_demetra_datestring` should probably be a date.

Comment (jonpage) [inline lib/series_seasonal_adjustment.rb]: If not we should refactor the functions above to return a Date.

File: lib/series_seasonal_adjustment.rb [hunk 4]
```diff
@@ -32,12 +32,12 @@ def set_factors(factor_application)
...
     last_demetra_date = Date.parse self.last_demetra_datestring
     factor_comparison_start_date = last_demetra_date << 12
```
Comment (jonpage) [inline lib/series_seasonal_adjustment.rb]: `last_demetra_date - 1.year` is a little easier to read

File: lib/series_data_adjustment.rb (hunk 1 above)
Comment (vcward) [inline lib/series_data_adjustment.rb]: Oh yeah, I must've done something weird when I pushed this one. I'll redo it

**Pull request closed without merging**